clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -296,6 +301,11 @@
                                         9 * 60 * 1000,
                                         Importance.MEDIUM,
                                         CommonClientConfigs.CONNECTIONS_MAX_IDLE_MS_DOC)
+                                .define(INTERCEPTOR_CLASSES_CONFIG,
+                                        Type.LIST,
+                                        "",
```

Comment:
@granthenke I did read that although I clearly misremembered part of it. Passing `null` seems like a better option, in that case, since any type can be null (as much as I dislike that). Passing an empty string for a LIST is surprising (I know we do that in some existing code, but I consider it legacy).